tests/tokenization/test_tokenization_utils.py

```diff
@@ -245,6 +245,74 @@ def test_padding_accepts_tensors(self):
         self.assertTrue(isinstance(batch["input_ids"], np.ndarray))
         self.assertEqual(batch["input_ids"].tolist(), [[0, 1, 2, tokenizer.pad_token_id], [0, 1, 2, 3]])
 
+    @require_tokenizers
```

Comment:
Thanks for adding tests! 💯  we can also merge fast & slow testing in one to help make changes in only one place in the future 😊 

```python
for tokenizer_class in [BertTokenizer, BertTokenizerFast]:
    with self.subTest(f"{tokenizer_class}"):
        tokenizer = tokenizer_class.from_pretrained("bert-base-uncased")
```

Reply:
ready :sunglasses: